vars/publishNotification.groovy

```diff
@@ -1,11 +1,13 @@
 void call(Map args = [:]) {
     text = ([
-        "${args.icon} ${JOB_NAME} [${BUILD_NUMBER}] ${args.message}",
-        "Build: ${BUILD_URL}",
-        args.manifest,
+        "${args.icon}", 
+        "JOB_NAME=${JOB_NAME}",
+        "BUILD_NUMBER=[${BUILD_NUMBER}]",
```

Comment:
Why use an array here? Can there be multiple build numbers in one notification?

Reply:
This is part of already existing code, I have just made more readable.

Reply:
Ohh.... I see. I wonder why is it an array though